internal/transform/trade.go

```diff
@@ -0,0 +1,210 @@
+package transform
+
+import (
+	"fmt"
+	"time"
+
+	ingestio "github.com/stellar/go/exp/ingest/io"
+	"github.com/stellar/go/xdr"
+	"github.com/stellar/stellar-etl/internal/utils"
+)
+
+// TransformTrade converts a relevant operation from the history archive ingestion system into a form suitable for BigQuery
+func TransformTrade(operationIndex int32, transaction ingestio.LedgerTransaction, ledgerCloseTime time.Time) ([]TradeOutput, error) {
```

Comment:
You would need to do ledger.LedgerHeader.ScpValue.CloseTime in order to get the close time. It seems easier to calculate the close time once and pass it to the transform function instead of passing the entire ledger to the transform function and repeating the calculation every time.